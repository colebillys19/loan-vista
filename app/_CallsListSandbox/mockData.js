/* eslint-disable prettier/prettier */
const MOCK_DATA = [
  {
    audio: 'deeder.mp3',
    date: '05/13/2021',
    dept: 'ABC',
    desc: 'bibendum imper magna yeet',
    id: 'cb50e5c5-fed2-43da-8a47-fc8f7df96222',
    rep: 'DEF',
    time: '06:00 AM',
  },
  {
    audio: 'hoydo.mp3',
    date: '05/14/2021',
    dept: 'GHI',
    desc: 'yo yoo yooo yoooo yooo yoo yo yoo yooo yoooo yooo yoo yo yoo yooo yoooo yooo yoo yo yoo yooo yoooo yooo yoo yo yoo yooo yoooo yooo yoo yo yoo yooo yoooo yooo yoo yo yoo yooo yoooo yooo yoo yo yoo yooo yoooo yooo yoo',
    id: 'cb50e5c5-fed2-43da-8a47-fc8f7df96223',
    rep: 'JKL',
    time: '07:00 AM',
  },
  {
    audio: 'shtude.mp3',
    date: '05/15/2021',
    dept: 'MNO',
    desc: 'magna bibendum imper amet',
    id: 'cb50e5c5-fed2-43da-8a47-fc8f7df96224',
    rep: 'PQR',
    time: '08:00 AM',
  },
  {
    audio: 'shtude.mp3',
    date: '05/15/2021',
    dept: 'MNO',
    desc: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
    id: 'cb50e5c5-fed2-43da-8a47-fc8fadf96224',
    rep: 'PQR',
    time: '08:00 AM',
  },
  {
    audio: 'shtude.mp3',
    date: '05/15/2021',
    dept: 'MNO',
    desc: 'magna bibendum imper amet',
    id: 'bb50e5c5-fed2-43da-8a47-fc8fadf96224',
    rep: 'PQR',
    time: '08:00 AM',
  },
];

export default MOCK_DATA;
