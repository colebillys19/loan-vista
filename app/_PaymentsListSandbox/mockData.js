const MOCK_DATA = [
  {
    accidentHealthInsurance: '$66.86',
    accruedLateCharge: '$70.42',
    date: '12/30/2020',
    desc: 'Principal Curtailment',
    effectiveDate: '01/02/2020',
    escrow: '$41.02',
    escrowBalance: '$38.82',
    fees: '$74.24',
    id: '29ba8a70-19d3-4ddf-b4ff-879b4f8c8fe6',
    interest: '$25.00',
    lifeInsurance: '$55.66',
    misc: '$45.28',
    otherTotal: '$11.32',
    principal: '$36.02',
    principalBalance: '$8.68',
    replacementReserve: '$97.76',
    restrictedReserve: '$15.48',
    suspense: '$83.46',
    time: '04:12 AM',
    total: '$14.93',
  },
  {
    accidentHealthInsurance: '$78.39',
    accruedLateCharge: '$26.40',
    date: '12/29/2020',
    desc: 'Principal Curtailment',
    effectiveDate: '01/02/2020',
    escrow: '$89.24',
    escrowBalance: '$10.64',
    fees: '$63.92',
    id: '748a3cec-9c60-4dd5-9979-5e70feaf7093',
    interest: '$54.94',
    lifeInsurance: '$20.55',
    misc: '$25.65',
    otherTotal: '$12.47',
    principal: '$41.33',
    principalBalance: '$54.80',
    replacementReserve: '$29.82',
    restrictedReserve: '$7.42',
    suspense: '$91.42',
    time: '08:16 PM',
    total: '$34.89',
  },
  {
    accidentHealthInsurance: '$31.63',
    accruedLateCharge: '$38.14',
    date: '12/26/2020',
    desc: 'Mortgage Payment',
    effectiveDate: '01/02/2020',
    escrow: '$12.79',
    escrowBalance: '$9.06',
    fees: '$17.95',
    id: 'fb74124b-3e78-4ab8-8178-10d18cf28b30',
    interest: '$47.38',
    lifeInsurance: '$15.12',
    misc: '$10.87',
    otherTotal: '$92.89',
    principal: '$62.03',
    principalBalance: '$15.99',
    replacementReserve: '$19.95',
    restrictedReserve: '$20.48',
    suspense: '$13.12',
    time: '09:32 PM',
    total: '$15.66',
  },
  {
    accidentHealthInsurance: '$39.74',
    accruedLateCharge: '$69.90',
    date: '12/25/2020',
    desc: 'Initial Escrow Deposit',
    effectiveDate: '01/02/2020',
    escrow: '$99.98',
    escrowBalance: '$46.00',
    fees: '$45.46',
    id: '152f341c-bba1-4813-aa53-26979a5dc80b',
    interest: '$75.10',
    lifeInsurance: '$74.43',
    misc: '$27.71',
    otherTotal: '$61.84',
    principal: '$46.51',
    principalBalance: '$49.87',
    replacementReserve: '$18.26',
    restrictedReserve: '$72.42',
    suspense: '$44.91',
    time: '05:53 PM',
    total: '$5.66',
  },
  {
    accidentHealthInsurance: '$90.40',
    accruedLateCharge: '$48.37',
    date: '12/16/2020',
    desc: 'Principal Curtailment',
    effectiveDate: '01/02/2020',
    escrow: '$40.58',
    escrowBalance: '$21.33',
    fees: '$3.24',
    id: '2d15a80f-1d71-49ff-90e1-82246574a4de',
    interest: '$74.84',
    lifeInsurance: '$77.91',
    misc: '$77.54',
    otherTotal: '$83.57',
    principal: '$77.68',
    principalBalance: '$29.92',
    replacementReserve: '$65.43',
    restrictedReserve: '$45.81',
    suspense: '$58.61',
    time: '05:57 AM',
    total: '$72.89',
  },
];

export default MOCK_DATA;
