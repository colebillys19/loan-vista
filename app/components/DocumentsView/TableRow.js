import React from 'react';
import T from 'prop-types';

import LinkButton from 'components/_shared/LinkButton';
import {
  StyledTableData,
  StyledTableRow,
} from 'components/_shared/GradientListTable';

const TableRow = ({ data, headers }) => (
  <StyledTableRow>
    {headers.map((header) => {
      if (header === 'pdf') {
        return (
          <StyledTableData key={header}>
            <LinkButton onClick={() => null} text="download" />
          </StyledTableData>
        );
      }

      return <StyledTableData key={header}>{data[header]}</StyledTableData>;
    })}
  </StyledTableRow>
);

TableRow.propTypes = {
  data: T.shape({
    'date sent': T.string,
    desc: T.string,
    from: T.string,
    pdf: T.string,
    'time sent': T.string,
    type: T.string,
  }),
  headers: T.arrayOf(T.string).isRequired,
};

export default TableRow;
