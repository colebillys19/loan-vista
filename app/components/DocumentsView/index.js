/**
 * DocumentsView
 * @description ...
 */

import React from 'react';
import T from 'prop-types';

import DocumentsList from 'components/DocumentsList';
import ListFilter from 'components/ListFilter';
import ConditionalRender from 'components/_base-ui/ConditionalRender';
import ListRenderFallback from 'components/_base-ui/ListRenderFallback';
import TableHeadBorder from 'components/_base-ui/TableHeadBorder';
import TabWrapper from 'components/_base-ui/TabWrapper';

const DocumentsView = ({
  dispatchFetchDocumentsData,
  documentsData,
  error,
  fetchParams,
  loading,
  sortValues,
}) => {
  const noListData = !loading && documentsData.length === 0;

  return (
    <TabWrapper aria-labelledby="documents-tab" id="documents-view">
      <ListFilter
        fetchParams={fetchParams}
        dispatchFetchData={dispatchFetchDocumentsData}
      />
      <TableHeadBorder hideBottom={loading || noListData} />
      <ConditionalRender
        Component={
          <DocumentsList
            documentsData={documentsData}
            dispatchFetchDocumentsData={dispatchFetchDocumentsData}
            loading={loading}
            sortValues={sortValues}
          />
        }
        FallbackComponent={<ListRenderFallback error={error} />}
        shouldRender={!error && !noListData}
      />
    </TabWrapper>
  );
};

DocumentsView.propTypes = {
  documentsData: T.array.isRequired,
  error: T.oneOfType([T.bool, T.string]).isRequired,
  fetchParams: T.object.isRequired,
  loading: T.bool.isRequired,
  dispatchFetchDocumentsData: T.func.isRequired,
  sortValues: T.object.isRequired,
};

export default DocumentsView;
