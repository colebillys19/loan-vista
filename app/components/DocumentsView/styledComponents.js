import styled from 'styled-components';

import { StyledTableHead } from 'components/_shared/GradientListTable';

export const DocumentsTableHead = styled(StyledTableHead)`
  & th:nth-of-type(1) {
    width: 13.4rem;
  }
  & th:nth-of-type(2) {
    width: 11.6rem;
  }
  & th:nth-of-type(3) {
    width: 8.8rem;
  }
  & th:nth-of-type(4) {
    width: 9.3rem;
  }
  & th:nth-of-type(6) {
    width: 7rem;
  }
`;
