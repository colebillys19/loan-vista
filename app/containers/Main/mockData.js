export const sidebarLoanSummaryData = [
  { label: 'Loan Type', value: 'FHA Residential' },
  { label: 'Status', value: 'Current' },
  { label: 'Principal Balance', value: '$234,456.78' },
  { label: 'Original Loan Amount', value: '$345,567,89' },
  { label: 'Investor Name', value: 'Nevada Housing' },
  { label: 'Investor Code', value: 'N03' },
  { label: 'Assumption Code', value: 'A' },
  { label: 'Assumption Date', value: '2/3/2020' },
  { label: 'Interest Rate', value: '5%' },
  { label: 'Origination Date', value: '5/6/2012' },
  { label: 'Maturity Date', value: '5/6/2042' },
  { label: 'Grace Days', value: '15' },
  { label: 'Lien Position', value: '1st Mortgage' },
];
