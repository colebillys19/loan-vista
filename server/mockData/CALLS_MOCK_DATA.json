[{"audio":"justo_in_hac.mp3","date":"2020-10-01","dept":"Khaki","details":"Praesent id massa id nisl venenatis lacinia.","rep":"#cb7","time":"14:29"},
{"audio":"tellus.mpeg","date":"2020-08-02","dept":"Crimson","details":"Nulla tempus.","rep":"#5d5","time":"0:23"},
{"audio":"ligula.avi","date":"2020-10-14","dept":"Blue","details":"Mauris sit amet eros.","rep":"#734","time":"23:49"},
{"audio":"sollicitudin_ut_suscipit.avi","date":"2020-08-13","dept":"Turquoise","details":"Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Mauris viverra diam vitae quam. Suspendisse potenti. Nullam porttitor lacus at turpis.","rep":"#26d","time":"4:31"},
{"audio":"bibendum.avi","date":"2020-09-16","dept":"Mauv","details":"Fusce posuere felis sed lacus. Morbi sem mauris, laoreet ut, rhoncus aliquet, pulvinar sed, nisl.","rep":"#fc3","time":"16:04"},
{"audio":"habitasse_platea_dictumst.mp3","date":"2020-09-18","dept":"Red","details":"Aenean fermentum. Donec ut mauris eget massa tempor convallis. Nulla neque libero, convallis eget, eleifend luctus, ultricies eu, nibh.","rep":"#f4f","time":"3:44"},
{"audio":"et.avi","date":"2020-06-26","dept":"Maroon","details":"Cras mi pede, malesuada in, imperdiet et, commodo vulputate, justo. In blandit ultrices enim.","rep":"#175","time":"17:23"},
{"audio":"accumsan_tellus.avi","date":"2020-08-10","dept":"Red","details":"Duis aliquam convallis nunc. Proin at turpis a pede posuere nonummy. Integer non velit.","rep":"#51e","time":"19:59"},
{"audio":"lectus_in_quam.mp3","date":"2020-11-19","dept":"Khaki","details":"Nam nulla. Integer pede justo, lacinia eget, tincidunt eget, tempus vel, pede.","rep":"#95d","time":"14:31"},
{"audio":"pellentesque_volutpat_dui.mpeg","date":"2020-10-01","dept":"Crimson","details":"Mauris sit amet eros. Suspendisse accumsan tortor quis turpis. Sed ante.","rep":"#9dc","time":"21:15"},
{"audio":"sed_vel.mpeg","date":"2020-10-09","dept":"Crimson","details":"Morbi non quam nec dui luctus rutrum. Nulla tellus. In sagittis dui vel nisl.","rep":"#534","time":"13:32"},
{"audio":"platea_dictumst.mp3","date":"2020-08-19","dept":"Purple","details":"Morbi a ipsum.","rep":"#8a8","time":"2:26"},
{"audio":"porta_volutpat_erat.avi","date":"2020-11-27","dept":"Violet","details":"Maecenas pulvinar lobortis est.","rep":"#d3b","time":"13:06"},
{"audio":"vestibulum.mp3","date":"2020-11-08","dept":"Orange","details":"Suspendisse potenti. Nullam porttitor lacus at turpis.","rep":"#8e9","time":"0:45"},
{"audio":"pellentesque.mp3","date":"2020-07-14","dept":"Crimson","details":"Vestibulum sed magna at nunc commodo placerat. Praesent blandit. Nam nulla.","rep":"#aa0","time":"9:44"},
{"audio":"morbi.avi","date":"2020-10-28","dept":"Blue","details":"Maecenas tincidunt lacus at velit. Vivamus vel nulla eget eros elementum pellentesque. Quisque porta volutpat erat.","rep":"#af6","time":"21:01"},
{"audio":"sit_amet_nulla.avi","date":"2020-10-20","dept":"Orange","details":"Vivamus vel nulla eget eros elementum pellentesque. Quisque porta volutpat erat.","rep":"#425","time":"4:51"},
{"audio":"integer_pede_justo.mp3","date":"2020-11-08","dept":"Indigo","details":"Quisque porta volutpat erat. Quisque erat eros, viverra eget, congue eget, semper rutrum, nulla. Nunc purus.","rep":"#a2c","time":"9:12"},
{"audio":"quis_augue_luctus.avi","date":"2020-07-29","dept":"Orange","details":"In hac habitasse platea dictumst.","rep":"#a89","time":"7:50"},
{"audio":"quam.avi","date":"2020-07-28","dept":"Puce","details":"Aliquam erat volutpat.","rep":"#c25","time":"16:54"},
{"audio":"odio_in.avi","date":"2020-12-02","dept":"Blue","details":"Vestibulum sed magna at nunc commodo placerat. Praesent blandit.","rep":"#c8b","time":"6:22"},
{"audio":"ipsum_ac_tellus.mp3","date":"2020-08-13","dept":"Fuscia","details":"Aliquam augue quam, sollicitudin vitae, consectetuer eget, rutrum at, lorem. Integer tincidunt ante vel ipsum. Praesent blandit lacinia erat.","rep":"#6c4","time":"13:13"},
{"audio":"dapibus_at_diam.mpeg","date":"2020-06-30","dept":"Orange","details":"Nulla justo. Aliquam quis turpis eget elit sodales scelerisque.","rep":"#b67","time":"6:07"},
{"audio":"integer_ac_leo.avi","date":"2020-11-14","dept":"Mauv","details":"Nam nulla. Integer pede justo, lacinia eget, tincidunt eget, tempus vel, pede.","rep":"#edb","time":"9:19"},
{"audio":"justo_maecenas_rhoncus.avi","date":"2020-07-28","dept":"Green","details":"Suspendisse potenti.","rep":"#fdc","time":"10:29"},
{"audio":"leo_rhoncus_sed.mov","date":"2020-08-28","dept":"Mauv","details":"Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Duis faucibus accumsan odio. Curabitur convallis. Duis consequat dui nec nisi volutpat eleifend.","rep":"#9dc","time":"5:47"},
{"audio":"tempus.mp3","date":"2020-09-24","dept":"Puce","details":"Morbi sem mauris, laoreet ut, rhoncus aliquet, pulvinar sed, nisl. Nunc rhoncus dui vel sem. Sed sagittis.","rep":"#40a","time":"5:35"},
{"audio":"penatibus.mpeg","date":"2020-10-28","dept":"Turquoise","details":"Cras mi pede, malesuada in, imperdiet et, commodo vulputate, justo. In blandit ultrices enim. Lorem ipsum dolor sit amet, consectetuer adipiscing elit.","rep":"#ed9","time":"8:30"},
{"audio":"nulla_elit_ac.mp3","date":"2020-07-04","dept":"Turquoise","details":"Morbi sem mauris, laoreet ut, rhoncus aliquet, pulvinar sed, nisl. Nunc rhoncus dui vel sem.","rep":"#b08","time":"22:32"},
{"audio":"ullamcorper_purus_sit.mp3","date":"2020-06-22","dept":"Crimson","details":"Aenean auctor gravida sem. Praesent id massa id nisl venenatis lacinia.","rep":"#47a","time":"15:30"}]