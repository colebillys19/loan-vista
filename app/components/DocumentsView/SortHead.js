/**
 * TableHead
 * @description ...
 */

import React from 'react';
import T from 'prop-types';

import ListSortButton from 'components/_shared/ListSortButton';
import {
  StyledTableHeader,
  StyledTableRow,
} from 'components/_shared/GradientListTable';

import { DocumentsTableHead } from './styledComponents';

const SortHead = ({
  dispatchFetchDocumentsData,
  headers,
  lastSortCol,
  lastSortOrder,
  sortLoading,
}) => {
  const handleSortClick = (header) => {
    const newSortCol = header;
    const newSortOrder =
      header === lastSortCol && lastSortOrder === 'desc' ? 'asc' : 'desc';

    dispatchFetchDocumentsData(newSortCol, newSortOrder);
  };

  return (
    <DocumentsTableHead>
      <StyledTableRow>
        {headers.map((header) => {
          if (['date sent', 'type', 'from'].includes(header)) {
            return (
              <StyledTableHeader key={header} scope="col">
                <ListSortButton
                  isActive={header === lastSortCol}
                  isAscending={
                    header === lastSortCol && lastSortOrder === 'asc'
                  }
                  loading={sortLoading === header}
                  onClick={() => handleSortClick(header)}
                  text={header}
                />
              </StyledTableHeader>
            );
          }

          return (
            <StyledTableHeader key={header} scope="col">
              {header}
            </StyledTableHeader>
          );
        })}
      </StyledTableRow>
    </DocumentsTableHead>
  );
};

SortHead.propTypes = {
  dispatchFetchDocumentsData: T.func.isRequired,
  headers: T.arrayOf(T.string).isRequired,
  lastSortCol: T.string.isRequired,
  lastSortOrder: T.string.isRequired,
  sortLoading: T.oneOfType([T.bool, T.string]).isRequired,
};

export default SortHead;
