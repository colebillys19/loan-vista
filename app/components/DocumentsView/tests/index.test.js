import React from 'react';
import { render } from 'react-testing-library';

import DocumentsView from '../index';

import { MOCK_DATA, MOCK_HEADERS } from './mockData';

const mockProps = {
  dispatchFetchDocumentsData: jest.fn(),
  documentsData: MOCK_DATA,
  lastSortCol: '',
  lastSortOrder: '',
  listHeaders: MOCK_HEADERS,
  nextPageToFetch: 1,
  scrollLoading: false,
  sortLoading: false,
};

const Component = <DocumentsView {...mockProps} />;

describe('<DocumentsView />', () => {
  global.addEventListener = jest.fn();
  global.removeEventListener = jest.fn();
  global.scrollTo = jest.fn();
  global.document = { scrollingElement: { scrollHeight: 0 } };
  global.innerHeight = 0;
  global.scrollY = 0;

  it('Expect not to log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(Component);
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(Component);
    expect(firstChild).toMatchSnapshot();
  });
});
