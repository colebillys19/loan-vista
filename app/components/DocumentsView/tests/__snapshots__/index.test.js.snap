// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DocumentsView /> Should render and match the snapshot 1`] = `
<div
  class="styledComponents__TableContainer-fhggrt-7 gRfYPI"
>
  <div
    class="GradientBorder-zuziiu-0 ldijcC"
  />
  <div
    class="GradientBorder-zuziiu-0 crasIP"
  />
  <div
    class="GradientBorder-zuziiu-0 kSpIlI"
  />
  <div
    class="GradientBorder-zuziiu-0 cgqNXZ"
  />
  <table
    class="styledComponents__StyledTable-fhggrt-1 fYoUQb"
  >
    <thead
      class="styledComponents__StyledTableHead-fhggrt-4 styledComponents__DocumentsTableHead-sc-721u5q-0 auRJh"
    >
      <tr
        class="styledComponents__StyledTableRow-fhggrt-6 fSdBuc"
      >
        <th
          class="styledComponents__StyledTableHeader-fhggrt-5 bDgOie"
          scope="col"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text styledComponents__StyledButton-sc-1723w1c-0 guxOHO MuiButton-disableElevation"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              date sent
              <span
                class="MuiButton-endIcon MuiButton-iconSizeMedium"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  style="height: 0.8rem; width: 0.8rem;"
                  viewBox="7 7 10 10"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M7 10l5 5 5-5z"
                    fill="rgb(136, 136, 136)"
                  />
                </svg>
              </span>
            </span>
          </button>
        </th>
        <th
          class="styledComponents__StyledTableHeader-fhggrt-5 bDgOie"
          scope="col"
        >
          time sent
        </th>
        <th
          class="styledComponents__StyledTableHeader-fhggrt-5 bDgOie"
          scope="col"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text styledComponents__StyledButton-sc-1723w1c-0 guxOHO MuiButton-disableElevation"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              type
              <span
                class="MuiButton-endIcon MuiButton-iconSizeMedium"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  style="height: 0.8rem; width: 0.8rem;"
                  viewBox="7 7 10 10"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M7 10l5 5 5-5z"
                    fill="rgb(136, 136, 136)"
                  />
                </svg>
              </span>
            </span>
          </button>
        </th>
        <th
          class="styledComponents__StyledTableHeader-fhggrt-5 bDgOie"
          scope="col"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text styledComponents__StyledButton-sc-1723w1c-0 guxOHO MuiButton-disableElevation"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              from
              <span
                class="MuiButton-endIcon MuiButton-iconSizeMedium"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  style="height: 0.8rem; width: 0.8rem;"
                  viewBox="7 7 10 10"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M7 10l5 5 5-5z"
                    fill="rgb(136, 136, 136)"
                  />
                </svg>
              </span>
            </span>
          </button>
        </th>
        <th
          class="styledComponents__StyledTableHeader-fhggrt-5 bDgOie"
          scope="col"
        >
          desc
        </th>
        <th
          class="styledComponents__StyledTableHeader-fhggrt-5 bDgOie"
          scope="col"
        >
          pdf
        </th>
      </tr>
    </thead>
    <tbody
      class="styledComponents__StyledTableBody-fhggrt-2 faBsMU"
    >
      <tr
        class="styledComponents__StyledTableRow-fhggrt-6 fSdBuc"
      >
        <td
          class="styledComponents__StyledTableData-fhggrt-3 ksTgJN"
        >
          date sent
        </td>
        <td
          class="styledComponents__StyledTableData-fhggrt-3 ksTgJN"
        >
          time sent
        </td>
        <td
          class="styledComponents__StyledTableData-fhggrt-3 ksTgJN"
        >
          type
        </td>
        <td
          class="styledComponents__StyledTableData-fhggrt-3 ksTgJN"
        >
          from
        </td>
        <td
          class="styledComponents__StyledTableData-fhggrt-3 ksTgJN"
        >
          desc
        </td>
        <td
          class="styledComponents__StyledTableData-fhggrt-3 ksTgJN"
        >
          <button
            class="MuiButtonBase-root styledComponents__StyledButton-sc-1xk0fom-0 fEIblC"
            tabindex="0"
            type="button"
          >
            download
          </button>
        </td>
      </tr>
    </tbody>
    <thead>
      <tr>
        <td
          class="styledComponents__TableFooterCell-fhggrt-8 gtQmvA"
        >
          hidden text
        </td>
      </tr>
    </thead>
  </table>
</div>
`;
